addons/website/views/snippets/s_contact_info.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_contact_info" name="Contact Info">
+    <section class="s_contact_info pt48 pb64 o_cc o_cc1">
```

Comment:
You can omit  `o_cc o_cc1` since it's default... no problem tho, if it was for me everything in website would have a color-preset defined, I'll let you choose  😉